omnigan/losses.py

```diff
@@ -254,30 +254,15 @@ def __init__(
     
     def __call__(self, prediction1, prediction2, target):
         loss_calc = BCELoss()
-        if self.opt.train.multi_level == True:
+        if self.opt.dis.ADVENT.multi_level == True:
```

Comment:
Details details but I would maybe call your loss attribute "self.opts" instead of "self.opt". I first thought you were initializing the loss with an optimizer but you're giving it the options dict so I think it would be clearer. :)